addons/event_booth_sale/tests/test_event_booth_sale.py

```diff
@@ -207,4 +207,6 @@ def test_event_booth_with_invoice(self):
             f"Invoice payment is in '{invoice.payment_state}' status while it should be '{in_payment_state}'.")
 
         self.assertEqual(booth.state, 'unavailable')
+        self.assertFalse(booth.is_paid)  # it is still pending reconciliation
+        invoice._invoice_paid_hook()
```

Comment:
It would be better to process the payment, and have the booth state updated automatically :point_up: